architecture/additional_concepts/sdn.adoc

```diff
@@ -9,94 +9,115 @@
 
 toc::[]
 
-OpenShift provides a unified cluster network for that enables communication
-between containers across the OpenShift cluster.  This cluster network is
-established and maintained by the "OpenShift SDN", which configures an overlay
-network using Open vSwitch (OVS).  Following is a detailed discussion of
+== Overview
+
+OpenShift uses a software-defined networking (SDN) approach to provide a unified
+cluster network for that enables communication between containers across the
```

Comment:
@adellape "for that".

Should it be provides a unified cluster network for enabling communication..." or "cluster network that enables communication..."?